data/examples/shaders/transition/luma-burn.effect

```diff
@@ -108,7 +117,7 @@ float4 PSDefault(VertData v_in) : TARGET {
 	//return transition;
 }
 
-technique Draw
+technique DrawDarkToLight
```

Comment:
I did not notice this at first, but changing the name here would break existing configurations. Please keep both the original and the new name for it.

Reply:
This and the next, I put at the beginning of the techniques to ensure they listed first. It doesn't line up as well for code review, but I think users will make more sense of it.